src/metatensor/models/cli/export_model.py

```diff
@@ -19,23 +22,28 @@ def _add_export_model_parser(subparser: argparse._SubParsersAction) -> None:
     parser.add_argument(
         "model",
         type=str,
-        help="Saved model which should be exprted",
+        help="Saved model which should be exported",
     )
     parser.add_argument(
         "-o",
         "--output",
         dest="output",
         type=str,
         required=False,
-        default="exported.pt",
         help="Filename of the exported model (default: %(default)s).",
     )
 
 
-def export_model(model: str, output: str) -> None:
+def export_model(model: str, output: str = "exported-model.pt") -> None:
     """Export a pretrained model to run MD simulations
 
     :param model: Path to a saved model
     :param output: Path to save the exported model
     """
-    raise NotImplementedError("model exporting is not implemented yet.")
+
+    # Load the model
+    loaded_model = load_model(model)
+
+    # Export the model
```

Comment:
Should we do some more checks here? Especially that the units are not None maybe?

Reply:
I think the idea was that, if no units are available, the numbers will be passed on to the engine as they are
@Luthaf is this true?

Reply:
Ahh yes I think you are right.

Reply:
yes!

Reply:
Although we might want to warn about this in here